frontend/components/ActionsDropdown/ActionsDropdown.tsx

```diff
@@ -22,6 +22,7 @@ interface IActionsDropdownProps {
   placeholder: string;
   onChange: (value: string) => void;
   disabled?: boolean;
+  greyed?: boolean;
```

Comment:
is this used?

Reply:
Meant to include removing that in the last cleanup commit - removed now, thanks